downstream/modules/platform/ref-guidelines-hosts-groups.adoc

```diff
@@ -4,7 +4,7 @@
 
 .Databases
 * When using an external database, ensure the `[database]` sections of your inventory file are properly set up.
-* To improve performance, do not place the database server and the automation controller server on the same network.
+* The database server should be on the same network or in the same datacenter as the {ControllerNameStart} server for performance reasons.
```

Comment:
Please don't shoot me, Robert! I'm looking at the use of the word "should" knowing that we are supposed to avoid the use of it, if at all possible. Sorry, it jumped out at me. 
```suggestion
*For the best performance, ensure that your database server and the {ControllerNameStart} server are in the same network or datacenter.
```
Here's what the IBM styles and standards say under Word usage: 
"Use with caution should   verb
Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”."